scripts/evaluation/stats.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
This is also something the `report` script should handle automtically. Would add that to the Issue.

Reply:
Added to https://github.com/symflower/eval-dev-quality/issues/301